docs/topics/con_automatic-tagging-of-an-application.adoc

```diff
@@ -8,8 +8,8 @@
 
 [role="_abstract"]
 
-{ProductShortName} can automatically add tags to the application based on the application analysis. Automatic tagging is especially useful when dealing with large portfolios of applications.
+{ProductShortName} automatically spawns language discovery and technology discovery tasks when adding an application to the *Application Inventory*. These tasks automatically add tags to the application. {ProductShortName} can automatically add tags to the application based on the application analysis. Automatic tagging is especially useful when dealing with large portfolios of applications.
 
```

Comment:
When the language discovery task is running, the tech discovery task and the analysis task (if triggered) will wait until the language discovery is complete. Afterward, these tasks will run in parallel.

Reply:
Fixed.